largescaleobserved.qmd

```diff
@@ -202,10 +177,10 @@ ratings = Table(
     leftjoin!(
       leftjoin!(
         DataFrame(ratings),
-        DataFrame(getproperties(movies, (:movieId, :nrtngs)));
+        select(DataFrame(movies), :movieId, :nrtngs);
```

Comment:
```suggestion
        select!(DataFrame(movies), :movieId, :nrtngs);
```

(won't really make any performance difference since it's freed almost immediately)